src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -8,146 +8,326 @@
 import org.kuali.kfs.coa.businessobject.Account;
 import org.kuali.kfs.coa.businessobject.AccountGlobal;
 import org.kuali.kfs.coa.businessobject.AccountGlobalDetail;
+import org.kuali.kfs.coa.businessobject.AccountType;
+import org.kuali.kfs.coa.businessobject.BudgetRecordingLevel;
+import org.kuali.kfs.coa.businessobject.Chart;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryType;
+import org.kuali.kfs.coa.businessobject.RestrictedStatus;
 import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.rice.krad.bo.GlobalBusinessObject;
+import org.kuali.rice.krad.bo.GlobalBusinessObjectDetailBase;
 import org.kuali.rice.krad.bo.PersistableBusinessObject;
 import org.kuali.rice.krad.service.BusinessObjectService;
+import org.kuali.rice.krad.util.ObjectUtils;
+import org.kuali.rice.location.framework.campus.CampusEbo;
 
-public class CuAccountGlobal extends AccountGlobal {
+import edu.cornell.kfs.module.cg.businessobject.InvoiceFrequency;
+import edu.cornell.kfs.module.cg.businessobject.InvoiceType;
+
+public class CuAccountGlobal extends AccountGlobal implements GlobalBusinessObject{
     
     private static final long serialVersionUID = 1L;
 
     protected String majorReportingCategoryCode;
-    protected MajorReportingCategory majorReportingCategory;
+    protected String accountPhysicalCampusCode;
+    protected Date accountEffectiveDate;
+    protected boolean accountOffCampusIndicator;
+    protected boolean closed;
+    protected String accountTypeCode;
+    protected String appropriationAccountNumber;
+    protected boolean accountsFringesBnftIndicator;
```

Comment:
For the applicable boolean fields that map to a matching field on the Account BO, I would suggest declaring them as Boolean object fields and requiring them to have non-null values in order for them to be included in the global updates. As an example, see the existing "pendingAcctSufficientFundsIndicator" account field, which is declared as a Boolean on the global version and which overrides the control type in the data dictionary.